tests/README.md

```diff
@@ -0,0 +1,30 @@
+Tests
+=====
+
+This is PrestaShop automated tests folder.
+
+Multiple type of tests are available:
+- `E2E` and `Selenium` folders contain end-to-end tests maintained by PrestaShop QA team
```

Comment:
I don't think it's a good idea to say they are maintained by a specific group. Anyone should be able to add/improve E2E tests, starting with the Core team.

```suggestion
- `E2E` and `Selenium` folders contain end-to-end tests
```